todos-app/backend/task_service/app/models/task.rb

```diff
@@ -1,2 +1,8 @@
 class Task < ApplicationRecord
+  validates :user_id, presence: true
+  validates :title, length: {maximum: 20}, presence: true
+  validates :description, length: {maximum: 200}, allow_nil: true
+  validates :status, inclusion: {in: [0, 1, 2]}, presence: true
+  validates :priority, inclusion: {in: [0, 1, 2]}, presence: true
```

Comment:
You can also use [enum](https://betterprogramming.pub/how-to-use-enums-in-rails-6-87600e292476)
If you have time, it might be interesting to read it.

Reply:
Thanks for the suggestion! I'll refactor my code with enum 👍